samples/LowAllocationWebServer/LowAllocationWebServerLibrary/RoutingTable.cs

```diff
@@ -29,7 +29,7 @@ public TRequestId GetRequestId(HttpRequestLine requestLine)
         public bool TryHandle(HttpRequest request, TcpConnectionFormatter response)
         {
             // TODO: this should not allocate new string
-            Utf8String requestUtf8 = request.Path.ToUtf8String(TextEncoder.Utf8);
+            Utf8String requestUtf8 = request.Path.ToUtf8String(TextEncoder.InvariantUtf8);
```

Comment:
Yeah, I had mixed feelings about this. TextEncoder used just 'Utf8' but EncodingData used 'InvariantUtf8'. The Invariant is there because the built-in symbol table happens to be invariant.

I'm fine with the implied invariant. I'll switch it to be just Utf8 (and Utf16).